packages/edit-site/src/store/test/reducer.js

```diff
@@ -95,13 +95,19 @@ describe( 'state', () => {
 
 		it( 'should close the inserter when opening the list view panel', () => {
 			expect(
-				blockInserterPanel( true, setIsListViewOpened( true ) )
+				blockInserterPanel( true, {
```

Comment:
@jsnajdr any thoughts about whether I should update the tests here differently? Personally I don't see why we need to import actions for the reducer's unit tests. Same goes for `setIsInserterOpened`.

Reply:
I think it's fine to import the actions in the reducer test and use them there. The same goes for selectors, it's a good idea to use them to check the state, like:
```js
expect( getSomething( state ) ).toBe( 'something' );
```
The reducer, actions and selectors are parts of one unit, the store, they are useless without each other, and there's little value in trying to separate them strictly.